dataprofiler/tests/profilers/test_int_column_profile.py

```diff
@@ -788,3 +788,35 @@ def test_profile_merge_bin_edges_indices(self):
         profile_2.update(data_2)
 
         profile_1 + profile_2
+        
+    def test_diff(self):
+        """
+        Makes sure the IntColumn Diff() works appropriately.
+        """
+        data = [2, 'not an int', 6, 4]
+        df = pd.Series(data).apply(str)
+        profiler1 = IntColumn("Int")
+        profiler1.update(df)
+
+        data = [1, 3]
+        df = pd.Series(data).apply(str)
+        profiler2 = IntColumn("Int")
+        profiler2.update(df)
+
+        # Assert the difference report is correct
+        expected_diff = {
+            'max': 3.0,
+            'mean': 2.0,
+            'min': 1.0,
+            'stddev': 0.5857864376269049,
+            'sum': 8.0,
+            'variance': 2.0
+        }
```

Comment:
should have both a negative and positive value in case we decide to utilize absolutes at some point or protect against.